README.md

```diff
@@ -329,6 +329,7 @@
 * [painlessjson](https://github.com/BlackEdder/painlessjson) - Convert between D types and std.json.
 * [std.data.json](https://github.com/dlang-community/std_data_json) - Phobos candidate for JSON serialization (based on Vibed)
 * [asdf](https://github.com/libmir/asdf) - Cache oriented string based JSON representation for fast read & writes and serialisation.
+* [jstruct](https://github.com/Hax-io/jstruct) - JSONify a struct and vice-versa
```

Comment:
I'm up for adding new projects, but this project has 0 stars on github and 16 downloads total on DUB. The functionality is also already provided by more popular packages such as painlessjson or mir-ion (which is also active in development and provides a standard (de)serialization framework that also works with a bunch of safety attributes and DIPs)

I don't think we should add this to the list here (it also only has 10 commits and only a few days of activity 2 months ago)

Reply:
Will update tomorrow evening 🤙👍